backend/pkg/connect/get_connectors.go

```diff
@@ -50,16 +69,32 @@ type ClusterConnectors struct {
 // ClusterConnectorInfo contains all information we can retrieve about a single
 // connector in a Kafka connect cluster.
 type ClusterConnectorInfo struct {
-	Name         string                     `json:"name"`
-	Class        string                     `json:"class"`
-	Config       map[string]string          `json:"config"`
-	Type         string                     `json:"type"`  // Source or Sink
-	Topic        string                     `json:"topic"` // Kafka Topic name
-	State        string                     `json:"state"` // Running, ..
-	TotalTasks   int                        `json:"totalTasks"`
-	RunningTasks int                        `json:"runningTasks"`
-	Trace        string                     `json:"trace,omitempty"`
-	Tasks        []ClusterConnectorTaskInfo `json:"tasks"`
+	Name         string                      `json:"name"`
+	Class        string                      `json:"class"`
+	Config       map[string]string           `json:"config"`
+	Type         string                      `json:"type"`  // Source or Sink
+	Topic        string                      `json:"topic"` // Kafka Topic name
+	State        connectorState              `json:"state"` // Running, ..
+	Status       connectorStatus             `json:"status"`
+	TotalTasks   int                         `json:"totalTasks"`
+	RunningTasks int                         `json:"runningTasks"`
+	Trace        string                      `json:"trace,omitempty"`
+	Errors       []ClusterConnectorInfoError `json:"errors"`
```

Comment:
@rikimaru0345 usually prefers to omit error props from the JSON if none are set. Maybe it's different for an array of errors, but he can speak for himself

Reply:
That's right for other properties, but I'd prefer if an array is empty don't remove it. leave it as `[]`.

If you already made the change, no problem either way.